IFC4x3/Properties/r/RQD_2o32BT3ZX4bO$w85z$rZ25/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RQD_2o32BT3ZX4bOw85zrZ25" Name="RQD" UniqueId="b20c22dd-0e38-4495-8ffa-205f7fd63085" PrimaryDataType="IfcPositiveRatioMeasure" />
```

Comment:
A more descriptive name would be welcome.

Reply:
Similar as a few other comments as well.

Reply:
Renamed.